src/swagger/trips.swagger.js

```diff
@@ -369,3 +369,37 @@
 *       '401':
 *             description: This request is not yours it is for another manager
 * */
+/**
+* @swagger
+* /api/v1/trips/search:
+*   get:
+*     tags:
+*       - Trips
+*     name: search a trip request
+*     summary: user can search for a trip request using different request attribute
+*     produces:
+*       - application/json
+*     consumes:
+*       - application/json
+*     parameters:
+*       - name: token
+*         in: header
+*         description: jwt token of the user
+*       - name: id
+*         in: query
+*       - name: location
+*         in: query
+*       - name: destination
+*         in: query
+*       - name: departureDate
+*         in: query
+*       - name: returnDate
+*         in: query
+*       - name: reason
+*         in: query
+*       - name: status
+*         in: query
+*     responses:
+*       '200':
+*             description: request found.
```

Comment:
You could add a 404 message too for this line, below 404.

Reply:
I added another response to reflect the response that will be sent when the user searches for a request